src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -45,10 +42,9 @@ public void onCommand(Member sender, CommandContext context) {
             return;
         }
         Member member = context.getGuild().getMember(userForInfo);
-        
         String status = "";
         String statusName = "";
-        EmbedBuilder builder = MessagingObjects.getClearThreadLocalEmbedBuilder();
+        EmbedBuilder builder = new EmbedBuilder(MessagingObjects.getMessageTypeEmbedBuilder(MessageType.INFO, sender.getUser()));
```

Comment:
Use the clear thread local embed builder

Reply:
Just use this instead: (Ryan and I have talked)
```suggestion
        EmbedBuilder builder = MessagingObjects.getMessageTypeEmbedBuilder(MessageType.INFO, sender.getUser());
```

Reply:
Using that just seems to break the way the pages work, the 1st page about the user is gone and is also replaced with roles